<ui version="4.0" >
 <class>udg::QCINEControllerBase</class>
 <widget class="QWidget" name="udg::QCINEControllerBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>164</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>1</number>
   </property>
   <property name="spacing" >
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="m_playToolButton" >
       <property name="text" >
        <string>Play</string>
       </property>
       <property name="icon" >
        <iconset resource="../main/main.qrc" >:/images/play.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="m_loopCheckBox" >
         <property name="text" >
          <string>Loop</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/repeat.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_boomerangCheckBox" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="text" >
          <string>Go-back</string>
         </property>
         <property name="icon" >
          <iconset resource="../main/main.qrc" >:/images/boomerang.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QSlider" name="m_velocityControl" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>60</number>
       </property>
       <property name="pageStep" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>10</number>
       </property>
       <property name="sliderPosition" >
        <number>10</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_velocityLabel" >
       <property name="text" >
        <string>10 img/sec</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main/main.qrc" />
 </resources>
 <connections/>
</ui>
